src/components/Reports/Reports.js

```diff
@@ -15,54 +15,47 @@ import { Helmet } from 'react-helmet';
 import { Loading } from '../Loading/Loading';
 import { addDays } from '../../utils';
 
-class Reports extends React.Component {
-  /**
-   * @param { Object } props
-   * @param { import('../../services/pure-di').AppServices } props.dependencies
```

Comment:
Maybe you would like to keep these annotations, they are useful for IntelliSense.

There should be already examples of function components with these annotations.

Reply:
ohh, thanks didn't know!